### PR TITLE
core: CoSi collective signatures

### LINKED ISSUE #450: implement CoSi on Trezor T
The CoSi signing method is available on T1 (so it exists in trezor-crypto), but not on TT. It should be added at some point.

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #450.
[[UI diff](https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/3382048498/artifacts/master_diff/index.html)]

cosi

### CODE REVIEW

File: core/src/apps/misc/cosi_commit.py
Comment (mmilata) [reply core/src/apps/misc/cosi_commit.py]: 3b988ab

File: core/src/apps/misc/cosi_commit.py
Comment (mmilata) [reply core/src/apps/misc/cosi_commit.py]: 3b988ab

File: core/src/apps/misc/cosi_commit.py
Comment (mmilata) [reply core/src/apps/misc/cosi_commit.py:17]: 3b988ab + b87dc8d
I'm guessing this will be used with `coin_name` == Bitcoin? Might be worth adding a test vector.

Comment (matejcik) [reply core/src/apps/misc/cosi_commit.py:17]: yep, coin name unset so Bitcoin, script type unset so the default. test vector would be nice.

**Review by matejcik — APPROVED** [review]
ACK from me, waiting for @andrewkozlik's review before merging.

**Review by andrewkozlik — APPROVED** [review]

**Pull request merged**
